src/helpers/customErrors.ts

```diff
@@ -22,3 +22,12 @@ export class MediaNotFoundError extends Error {
     this.name = 'MediaNotFoundError';
   }
 }
+
+export class IMDbIDNotFoundError extends MediaNotFoundError {
```

Comment:
I think it would be better to extend the error class?

Reply:
It makes more sense now, can you look again?